src/software/sensor_fusion/sensor_fusion_test.cpp

```diff
@@ -35,6 +38,9 @@ class SensorFusionTest : public ::testing::Test
     World test_world;
     std::unique_ptr<TbotsRobotMsg> tbots_robot_msg_id_1;
     std::unique_ptr<TbotsRobotMsg> tbots_robot_msg_id_2;
+    std::unique_ptr<SSL_Referee> ref_ind_yellow;
+    std::unique_ptr<SSL_Referee> ref_ind_blue;
```

Comment:
`ref_ind` wasn't immediately obvious to me that it meant indirect free kick. Could we make these variables a bit more verbose?

Reply:
fixed